File: python/frogJump.py
# you can use print for debugging purposes, e.g.
# print "this is a debug message"

def solution(X, Y, D):
    # write your code in Python 2.7
    dist = Y - X
    jumps = dist / D
    mod = dist % D
    if mod > 0:
        jumps += 1;
    return jumps
    pass

File: cpp/frogJump.cpp
// you can use includes, for example:
// #include <algorithm>

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

int solution(int X, int Y, int D) {
    // write your code in C++11
    int distance  = Y - X;
    int result = distance / D + ((distance % D) > 0 ? 1 : 0 ); 
    
    return result;
}

